src/MobxPromise.ts

```diff
@@ -112,10 +112,9 @@ export class MobxPromiseImpl<R>
 				if (status !== 'complete')
 					return status;
 
-		let status = this.internalStatus; // force mobx to track changes to internalStatus
-		if (this.latestInvokeId != this.invokeId)
-			status = 'pending';
```

Comment:
this bit is no longer necessary because we call `setPending` synchronously in `invokeResult`